Source/MedicalInfo/MainTabWindow_Medical.cs

```diff
@@ -232,13 +232,40 @@ public override void DoWindowContents(Rect rect)
             var offset = true;
 
             // extra 15f offset for... what? makes labels roughly align.
-            var colWidth = (rect.width - x - 15f) / CapDefs.Count;
-            for (var i = 0; i < CapDefs.Count; i++)
+            var colWidth = (rect.width - x - 15f) / (CapDefs.Count + 2);
+            for (var i = 0; i < 2; i++)
```

Comment:
Also, I think I'd prefer if we merged health & operations, save some screen space. The labels for CapDefs are getting squished on my resolution.